src/client/Microsoft.Identity.Client/Platforms/Features/DesktopOS/Kerberos/TicketCacheReader.cs

```diff
@@ -8,12 +8,10 @@
 
 namespace Microsoft.Identity.Client.Platforms.Features.DesktopOs.Kerberos
 {
-
-#pragma warning disable 618 // This workaround required for Native Win32 API call
     /// <summary>
     /// Helper class to check Kerberos Ticket in user's Ticket Cache.
     /// </summary>
-    public class TicketCacheReader : IDisposable
+    internal class TicketCacheReader : IDisposable
```

Comment:
Isn't this a breaking change?

Reply:
Looks like these were supposed to be internal and no one's supposed to be using it. KerberosSupplementalTicketManager has the public logic.

Reply:
+1 @neha-bhargava and @pmaytak - I forgot to update this.